src/components/Answers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Answers = (props) => {
```

Comment:
Little weird to call the component and file `Answers` rather than just `Answer`, as it's only in charge of rendering a single answer. Let's rename this to `Answer`.